src/overlays/ovl_i6/fox_ve2.c

```diff
@@ -0,0 +1,4132 @@
+extern Gfx D_C002F00;
+extern Gfx D_C0043D0;
+extern Gfx D_C004860;
+extern Gfx D_C015740;
+extern Gfx D_C022520;
+
+static Vec3f D_i6_801A67C4 = { 40.0f, 0.0f, -10.0f };
+static f32 D_i6_801A67D0 = 180.0f;
+static f32 D_i6_801A67D4 = 0.0f;
+static f32 D_i6_801A67D8 = 0.0f;
+static Vec3f D_i6_801A67DC = { 40.0f, 0.0f, 10.0f };
+static f32 D_i6_801A67E8 = 0.0f;
+static f32 D_i6_801A67EC = 0.0f;
+static f32 D_i6_801A67F0 = 0.0f;
+static Vec3f D_i6_801A67F4 = { 0.0f, 0.0f, 0.0f };
+
+s32 func_i6_801917F0(s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3f* rot, void* data) {
+    Vec3f sp94;
+    Vec3f sp88;
+    f32 scale;
+    Boss* boss = (Boss*) data;
+
+    sp94 = D_i6_801A67F4;
+    scale = 1.0f;
+    if ((limbIndex >= 13) && (limbIndex <= 50)) {
+        if (boss->actionState < 2) {
+            *dList = NULL;
+        } else {
+            RCP_SetupDL(&gMasterDisp, 0x1D);
+            gSPTexture(gMasterDisp++, 6600, 6600, 0, G_TX_RENDERTILE, G_ON);
+            gSPSetGeometryMode(gMasterDisp++, G_TEXTURE_GEN);
+            if (((limbIndex >= 23) && (limbIndex <= 32)) || (limbIndex == 50)) {
+                if (boss->swork[5] < 0) {
+                    *dList = NULL;
+                } else if (boss->swork[3] & 1) {
+                    RCP_SetupDL(&gMasterDisp, 0x1B);
+                    GPC(255, 0, 0, 255);
+                }
+            } else if (boss->swork[4] < 0) {
+                *dList = NULL;
+            } else if (boss->swork[2] & 1) {
+                RCP_SetupDL(&gMasterDisp, 0x1B);
+                GPC(255, 0, 0, 255);
+            }
+        }
+    } else if (!(boss->timer_05C & 1)) {
+        RCP_SetupDL(&gMasterDisp, 0x1D);
+    } else {
+        RCP_SetupDL(&gMasterDisp, 0x1B);
+        GPC(255, 0, 0, 255);
+    }
+
+    switch (limbIndex) {
+        case 59:
+            if (boss->swork[18] == 1) {
+                *dList = &D_C004860;
+            }
+            if (boss->swork[18] == 2) {
+                *dList = NULL;
+            }
+            break;
+        case 3:
+            rot->z -= boss->vwork[4].x;
+            rot->y -= boss->vwork[4].y;
+            if (boss->swork[0] & 1) {
+                RCP_SetupDL(&gMasterDisp, 0x1B);
+                GPC(255, 0, 0, 255);
+            }
+            if (boss->swork[11] != 0) {
+                *dList = &D_C0043D0;
+                RCP_SetupDL(&gMasterDisp, 0x15);
+            }
+            scale = boss->fwork[17];
+            break;
+        case 4:
+            rot->z -= boss->vwork[5].x;
+            rot->y -= boss->vwork[5].y;
+            if (boss->swork[1] & 1) {
+                RCP_SetupDL(&gMasterDisp, 0x1B);
+                GPC(255, 0, 0, 255);
+            }
+            if (boss->swork[12] != 0) {
+                *dList = &D_C015740;
+                RCP_SetupDL(&gMasterDisp, 0x15);
+            }
+            scale = boss->fwork[17];
+            break;
+        case 5:
+            if (boss->swork[17] == 1) {
+                *dList = &D_C002B20;
```

Comment:
```suggestion
                *dList = D_C002B20;
```
These guys should be Gfx[]